parsers/BO.py

```diff
@@ -77,10 +56,24 @@ def fetch_production(
         if target_datetime is not None and hour < 24:
             timestamp = timestamp.replace(hour=hour - 1)
 
-        hour_resp = template_response(zone_key, timestamp.datetime, "cndc.bo")
-        hour_resp["production"]["unknown"] = thermo
-        hour_resp["production"]["hydro"] = hydro
-        hour_resp["production"]["wind"] = wind
+        # NOTE: that thermo includes gas + oil are mixed, so this is unknown for now
+        modes_extracted = [hydro, solar, wind]
```

Comment:
`_bagasse` should be fed into `biomass` (It's leftovers from sugar cane production)

Reply:
Fixed in https://github.com/tmrowco/electricitymap-contrib/pull/3553/commits/cbb21eed8d3483a3874278324565b01f11292a9c